### DISCUSSION TITLE
Anyone have any recommendations on how to add GA click Event tracking

### DISCUSSION THREAD

**Discussion by munts** [flyntwp/flynt#453] [Q&A] [answered]
I thought I could add something like 
$('.memberCta a').on('click', function () {
      var member = $('h3.content-title').html()
      ga('send', 'event', 'Featured Banner', 'Click', member)
    })

to the main.js, but I can't get my build to complete because I keep getting 'error  'ga' is not defined  no-undef'

I looked into adding it as a new bind()  event to the scripts.js for FeaturedGoogleAnalytics, but I wasn't able to get that to work either.

Should I add some sort of config to the build-config.js to allow this to pass or does anyone know if there a better way to do this?

Thanks,
Scott

**Comment by timohubois** [comment, marked as answer]
@munts you can define ga as global, as did inside [FeatureAdminComponentScreenshots](https://github.com/flyntwp/flynt/blob/6ba22b9fc9d5164c90e17f0dcbead6ff8d3fcd23/Components/FeatureAdminComponentScreenshots/admin.js#L1) to prevent this error. But you should be sure that ga will be loaded before and is available before the click function run.
